Gem/Code/Source/AutoGen/NetworkMatchComponent.AutoComponent.xml

```diff
@@ -22,6 +22,7 @@
                      Count="MaxSupportedPlayers" IsPublic="true" IsRewindable="false" IsPredictable="true"
                      ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="true"
                      Description="The current states of players in-game resources"/>
+    <NetworkProperty Type="uint16_t" Name="PlayerCount" Init="0" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="true" IsPredictable="false" ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="true" Description="The remaining time in the round in seconds" />
```

Comment:
Wrong descriptions. But we should actually change that in stab not during a merge :p